content/getting-started/yaml.md

```diff
@@ -143,11 +143,11 @@ Note that `mac_pro`, `linux`, and `linux_x2` are only available for teams and us
 
 ### Environment
 
-`environment:` contains all the environment variables and enables to specify the version of Flutter, Xcode, CocoaPods, Node, npm and Ruby used for building. This is also where you can add credentials and API keys required for [code signing](../code-signing-yaml/signing). Make sure to [encrypt the values](../building/encrypting) of variables that hold sensitive data. 
+`environment:` specifies the environment variables and build machine software versions.
 
-{{<notebox>}}
-Using a non-default version of Ruby for macOS builds will increase the time of your `Preparing build machine` step significantly. 
-{{</notebox>}}
+##### Workflow environment variables
```

Comment:
this could be ####

Reply:
I prefer the `#####` otherwise the headings are too big

Reply:
After another look, I decided to change them to `####` for logical consistency. They're not too big.